src/main/java/the_warlord/powers/NextTurnSyrettePower.java

```diff
@@ -0,0 +1,55 @@
+package the_warlord.powers;
+
+import basemod.interfaces.CloneablePowerInterface;
+import com.megacrit.cardcrawl.actions.common.MakeTempCardInHandAction;
+import com.megacrit.cardcrawl.actions.common.ReducePowerAction;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.localization.PowerStrings;
+import com.megacrit.cardcrawl.powers.AbstractPower;
+import the_warlord.WarlordMod;
+import the_warlord.cards.warlord.Syrette;
+
+public class NextTurnSyrettePower extends AbstractPower implements CloneablePowerInterface {
+    public static final String POWER_ID = WarlordMod.makeID("NextTurnSyrettePower");
+    private static final PowerStrings powerStrings = CardCrawlGame.languagePack.getPowerStrings(POWER_ID);
+    public static final String NAME = powerStrings.NAME;
+    public static final String[] DESCRIPTIONS = powerStrings.DESCRIPTIONS;
+
+    public NextTurnSyrettePower(AbstractCreature owner, int amount) {
+        name = NAME;
+        ID = POWER_ID;
+        this.owner = owner;
+        this.amount = amount;
+        type = PowerType.BUFF;
+        updateDescription();
+        isTurnBased = true;
+        // TODO: add proper art
+        loadRegion("explosive");
+    }
+
+    @Override
+    public void updateDescription() {
+        if(this.amount == 0) { description = DESCRIPTIONS[0]; }
+        else{ description = DESCRIPTIONS[1] + this.amount + DESCRIPTIONS[2]; }
+    }
```

Comment:
There's a different way to format this, which is pretty clean:
        this.description = amount == 0 ? DESCRIPTIONS[0] : String.format(DESCRIPTIONS[1], this.amount);

Using String.format in this way allows you to inject variable values in the middle of a string, so you don't have to concatenate multiple DESCRIPTIONS bits. In WarlordMod-Power-Strings.json, you can refer to this construction like this: 
    "DESCRIPTIONS": [
      "Add a Syrette to your hand at the start of next turn",
      "Add a Syrette to your hand at the start of next #b%1$s turns."
    ]

To break down the monster that is #b%1$s:
#b - make it blue, just like the base game does for such numbers
%...s - format as a String
1$ - replace with the argument value at index 1. In the case of String.format(DESCRIPTIONS[1], this.amount), 1$ refers to the this.amount